_SpecRunner.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>Jasmine Spec Runner</title>
-  <link rel="shortcut icon" type="image/png" href=".grunt\grunt-contrib-jasmine\jasmine_favicon.png">
```

Comment:
This file should be generated on run-time ideally, but let it be there.

Reply:
That's weird that it changed that line, I didn't touch that file. I don't think this file exists in LBLD, I wonder if it's actually using it at all? The `jasmine.json` file existed but was totally ignored by Jasmine.

Reply:
Yes I was right... I can delete that file and it doesn't affect the tests at all. It is being generated at runtime.

Reply:
Which is to say, it gets re-generated at runtime.